src/content/blog/2024/office_hours.mdx

```diff
@@ -0,0 +1,57 @@
+---
+title: 'Introducing nf-core office hours'
+subtitle: 'A new initiative - weekly drop-in sessions to connect and chat.'
+headerImage: 'https://images.unsplash.com/photo-1545239351-ef35f43d514b'
+headerImageAlt: 'Person using macbook pro and holding cappuccino'
+announcement:
+    text: 'Introducing nf-core office hours'
+    start: 2024-06-13T16:14:00+01:00
+    end: 2024-06-15T16:14:00+01:00
+pubDate: 2024-06-13T16:14:00+01:00
+authors:
+    - 'christopher-hakkaart'
+label:
+    - 'community'
+---
+
+# nf-core office hours
+
+Slack is great, but sometimes you can't beat sitting down and chatting to someone about your code over a cup of coffee.
+We are thrilled to introduce _nf-core office hours_, weekly drop-in sessions designed for nf-core developers seeking an opportunity to connect and chat.
```

Comment:
See above, it seems a bit vauge at the moment

Reply:
I thought it was fine as it was tbh, but the chatgpt one is also fine.

Reply:
```suggestion
We are thrilled to introduce _nf-core office hours_, weekly drop-in sessions designed for nf-core developers who are looking to get unstuck with their coding, connect with fellow developers, or collaborate on projects.
```

Reply:
```suggestion
We are thrilled to introduce _nf-core office hours_, weekly drop-in sessions designed for nf-core developers seeking an opportunity to connect, chat, and work together.
```